### ISSUE TITLE
Certain Closures Not Being Fixed

### ISSUE THREAD

**Issue by GrahamCampbell** [FriendsOfPHP/PHP-CS-Fixer#1094]
I'd expect the following:

```php
<?php

$function = function($foo) { return 'foo'; };
```

to be fixed to:

```php
<?php

$function = function ($foo) {
    return 'foo';
};
```

but instead, it's fixed to:

```php
<?php

$function = function ($foo) { return 'foo'; };
```

ie, there's basically no change (apart from the movement of the bracket just after "function").

**Comment by keradus** [comment]
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.5/Symfony/CS/Fixer/PSR2/BracesFixer.php#L162

Feel free to enhance fixer to support lambdas as well

**Comment by GrahamCampbell** [comment]
@keradus Unfortunately, I've got no time now, I'm just putting this here so that either someone else will fix it, or I will eventually fix it. :)

**Comment by keradus** [comment]
OK. I remember there were some problems with lambdas.

**Comment by rodolforg** [comment]
Just another case:
$app->get('/foo', function (ServerRequestInterface $request, ResponseInterface $response) { return $response->withStatus(200);});

**Comment by keradus** [comment]
closed by #1894
